Modules/Scripted/EditorLib/HelperBox.py

```diff
@@ -584,8 +582,6 @@ def updateStructures(self,caller=None, event=None):
     self.setMergeButton.setDisabled(not self.master)
 
     # reset to a fresh model
-    self.brushes = []
-    self.items = []
```

Comment:
Indeed. See commit message for more details: https://github.com/jcfr/Slicer/commit/a75da9c562778aeb0c76a38a6ae14ceb5e75439f